### ISSUE TITLE
when  LV_IMAGE_ALIGN_TILE is used,  lv_image is incorrect in vg_lite drawing enabled

### ISSUE THREAD

**Issue by huohongpeng** [kisvegabor/lvgl#7204]
### LVGL version

v9.2

### What happened?

I used lv_image to draw image in  LV_IMAGE_ALIGN_TILE and vg_lite enabled,  only one image was drawed , not tile more image.

**Comment by liamHowatt** [comment]
Looks like image tiling is not supported with VG Lite yet.

See these generated ref images for the same test.

- Drawn by software renderer: https://github.com/lvgl/lvgl/blob/4aa74e15c3f21ff1a53fb5b699910c6dcf901f44/tests/ref_imgs/widgets/image_tile.png
- Drawn by VG Lite renderer: https://github.com/lvgl/lvgl/blob/4aa74e15c3f21ff1a53fb5b699910c6dcf901f44/tests/ref_imgs_vg_lite/widgets/image_tile.png

**Comment by kisvegabor** [comment]
cc @FASTSHIFT @nicusorcitu

**Comment by nicusorcitu** [comment]
@cosmindanielradu19

**Comment by cosmindanielradu19** [comment]
Hi, we do support tiled images, and the benchmark results looks fine on our side.

https://github.com/lvgl/lvgl/blob/master/src/draw/nxp/vglite/lv_draw_vglite_img.c#L178

**Comment by liamHowatt** [comment]
Right, if I recall correctly, tiling works on-device but not in the simulated environment.

**Comment by kisvegabor** [comment]
In the ThorVG implementation tiling really doesn't work. See the test images:
- ThorVG/VGLite API: https://github.com/lvgl/lvgl/blob/master/tests/ref_imgs_vg_lite/widgets/image_tile.png
- SW render: https://github.com/lvgl/lvgl/blob/master/tests/ref_imgs/widgets/image_tile.png

**Comment by kisvegabor** [comment]
@FASTSHIFT should it be fixed?

**Comment by FASTSHIFT** [comment]
> @FASTSHIFT should it be fixed?

Yes, I'll take some time to look at it.

**Comment by kisvegabor** [comment]
@FASTSHIFT Is it on your radar?

**Comment by FASTSHIFT** [comment]
The image width of vg-lite needs to be aligned to 16px. For images that are not aligned to 16px, you may see non-image data used to fill the alignment. This may not work well in tiled mode and is not supported yet.
